### PR TITLE
Move from a single static html file to Flask

### PR DESCRIPTION
**Pull request by mgeraci**
* add flask and its requirements
* add a single route, for the home page
* move the current index.html file to the `templates` folder
* update webpack to output to the `static` folder

### CODE REVIEW

File: lemonparty2018.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+app = Flask(__name__)
+
+
+@app.route('/')
+def index():
+    return render_template('index.html')
+
+
+if __name__ == '__main__':
+    app.run(debug=True, host='0.0.0.0')
```
Comment (katur) [inline lemonparty2018.py]: We should make `debug` configurable (in a local settings file, or an environment variable)

File: README.md [hunk 2]
```diff
@@ -1,12 +1,26 @@
 # Lemon Party 2018
 Let's see if this goes off with a hitch...
 
+## Backend
 
-## Static assets
+The back end is written in Python, and served with Flask. To get started,
```
Comment (katur) [inline README.md]: The header of this section has backend as one word

File: README.md [hunk 3]
```diff
@@ -1,12 +1,26 @@
...
+
+## Frontend
 
 The CSS is written using SASS, and the JS is written in ES6. Both are managed
 with webpack. Assuming you already have node and npm installed, run `npm install`
 to install the JS packages required for this app.
 
 Then to start webpack watching and compiling the files, run `npm run watch`. To
-compile the minified files for use in production, run `npm run build`.
+compile the minified files for use in production, run `npm run build`. TODO:
```
Comment (katur) [inline README.md]: If we are using yarn, this readme section needs to be updated

Comment (mgeraci) [reply README.md]: I updated this in the `install` part, but we will still use npm for compiling. Yarn just replaces the installer.

File: README.md [hunk 4]
```diff
@@ -1,12 +1,36 @@
...
+* `pip install -r requirements.txt`
+* add a file in the root of the project called `localsettings.py`. example
+  contents are listed below.
+* run the server with `python lemonparty2018.py`
+
+Then you should be able to load localhost:5000 and see the site.
+
+### `localsettings.py`
+
+`localsettings` stores information specific to a particular enbironment. Here is
```
Comment (katur) [inline README.md]: typo here (https://en.wikipedia.org/wiki/Lord_Byron)

**Comment by katur** [comment]
MA!

File: README.md (hunk 4 above)
Comment (mgeraci) [reply README.md]: 👍

**Pull request merged**